mappings/net/minecraft/datafixers/fixes/PointOfInterestRenameFix.mapping

```diff
@@ -1,2 +1,2 @@
-CLASS net/minecraft/class_4620 net/minecraft/datafixers/fixes/PointOfInterestRenameFIx
+CLASS net/minecraft/class_4620 net/minecraft/datafixers/fixes/PointOfInterestRenameFix
```

Comment:
Note: This change has a conflict that needs special handling when merged with #1015

Reply:
This now needs fixing.

Reply:
Fixed.